Nosenti/Innocent-Ingabire-portfolio
# 'ch u ifolder'

#### What does this PR do?
 Create a UI folder and put all folders 
#### Description of Task to be completed
Created a new folder and linked pages that I finished building 
#### How should this be manually tested?
No testing is required but if one wants to check, they can download the folder and see the arrangement of the files.
#### What are the relevant stories?
N/A
#### Screenshots (if appropriate)
